reactor/presence.sls

```diff
@@ -1,6 +1,8 @@
+# actions to run when a minion disappears or appears in the master
+
 # remove unused keys for minions that are not present anymore
-{% for minion_id in data['lost'] %}
-remove_unused_keys_for_{{ minion_id }}:
-   wheel.key.delete:
-     - match: {{ minion_id }}
-{% endfor %}
+# {% for minion_id in data['lost'] %}
```

Comment:
Yeah, probably... I was not sure what do do here. Not sure if we should remove the key when a node is considered as _lost_ from Salt...